src/ext/UtilExtension/wixext/Data/tables.xml

```diff
@@ -134,6 +134,8 @@
                 category="text" description="Domain half of user account to secure"/>
         <columnDefinition name="User" type="string" length="255" primaryKey="yes" modularize="property"
                 category="text" description="Username half of user account to secure"/>
+        <columnDefinition name="Inheritable" type="number" length="1" modularize="column"
```

Comment:
Unfortunately, cannot add columns to a table. This will break patching. It appears this table is old and doesn't have the generic Attributes column we stared adding to somewhat future proof ourselves for these sorts of changes.

I'm afraid this change will need to move to WiX v4 only.